cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,35 @@
 
 describe('Book Store app', () => {
   before(() => {
-    
+
+    cy.viewport(1200, 1200)
+    cy.visit('https://demoqa.com/login')
   });
 
-  it('', () => {
-    
+  it('login, add to cart, delete from cart', () => {
+    cy.get('#userName').type('lexif');
+    cy.get('#password').type('Lexif@2022');
+    cy.get('#login').click();
+    cy.get('[id="userName-value"]').should('contain', 'lexif');
+    cy.url().
+    should('include', 'profile');
+    cy.get(':nth-child(6) > .element-list > .menu-list > #item-2').click();
```

Comment:
find a better and shorter selector

Reply:
didn't find a better solution. getting error with other variants